curated_transformers/models/roberta/_hf.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Mapping
+import re
+from torch import Tensor
+from torch.nn import Parameter
+
+from .config import RobertaConfig
+from ..hf_util import _merge_qkv
+
+
+def convert_hf_config(hf_config: Any) -> RobertaConfig:
+    # WARNING: seem like we cannot get the max length from the model? Maybe
+    # we should do the max position embeddings minus 2?
```

Comment:
Yeah, I think that's part of the tokenizer for some odd reason. We should be able to deserialize its config along with the model's, I think?

Reply:
We can still pass the tokenizer config as an `Optional`, use its `model_max_length` attribute when available and falling back to the position embeddings when it's missing? That seems a bit more robust than always using the latter.

Reply:
After internal discussion, we'll keep the current implementation as the model should be seen as the source of truth for `max_length`, and to reduce coupling between models and their tokenizers.